sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -153,33 +153,33 @@ def setup_cache(model, shard_count):
             )
             page_dim = torch.export.Dim("page")
 
-            dynamic_shapes = [
-                {0: page_dim} for _ in range(llama_config.pipeline_parallelism_size)
-            ]
+            pipeline_parallelism_size = len(cache_state)
+            tensor_parallelism_size = (
+                cache_state[0].shard_count
```

Comment:
Once it starts spreading across multiple lines like this the benefits of not having a regular if really disappear. I would just do

```
pipeline_parallelism_size = 1
if isinstance(cache_state[0], ShardedTensor):
  pipeline_parallelism_size = cache_state[0].shard_count
```